### PR TITLE
Update manage-windows-devices.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the URL as from august 1st the current Intune Azure Portal experience will be retired

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2466

File: microsoft-365/business/manage-windows-devices.md [hunk 1]
```diff
@@ -45,8 +45,7 @@ See [Synchronize domain users to Microsoft](manage-domain-users.md) for the step
 
 ## 1. Verify MDM Authority in Intune
 
-Go to portal.azure.com and on the top of the page search for Intune.
-On the Microsoft Intune page, select **Device enrollment** and on the **Overview** page make sure **MDM authority** is **Intune**.
+Go to [Endpoint Manager](https://endpoint.microsoft.com/#blade/Microsoft_Intune_Enrollment/EnrollmentMenu/overview) and On the Microsoft Intune page, select **Device enrollment** and on the **Overview** page make sure **MDM authority** is **Intune**.
```
Comment (JohanFreelancer9) [inline microsoft-365/business/manage-windows-devices.md]:
```suggestion
Go to [Endpoint Manager](https://endpoint.microsoft.com/#blade/Microsoft_Intune_Enrollment/EnrollmentMenu/overview) and on the Microsoft Intune page, select **Device enrollment**, then on the **Overview** page, make sure **MDM authority** is **Intune**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks

**Pull request merged**